nltk/translate/util.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+
+def jacknifing(score_list, averaging=True):
+        ''' This is a averaging function for calculating ROUGE score
+    when multiple references are present. This technique has been
+    referred to as the Jacknifing technique in the original paper.
+    score_list = list of scores over which averaging occurs
+    '''
+        if(len(score_list) == 1):
+            return np.mean(score_list)
+        elif((len(score_list) > 1) and (averaging is False)):
+            return score_list
+        else:
+            average = []
+            for i in score_list:
+                dummy = [j for j in score_list if i != j]
+                average.append(max(dummy))
+            return(np.mean(average))
+
+
+def lcs(X, Y, m, n):
+        '''This function returns the length of the
+        longest common subsequence(LCS) between two
+        given strings and also the LCS.
+        X = list containing all the tokens of the first string
+        Y = list containing all the tokens of the second string
+        m = no. of tokens of the first string
+        n = no.of tokens of the second string
+
+        Note: Tokenization of the strings can be done at both
+        character-level and word-level.
```

Comment:
Please use proper conventions to explain the arguments, e.g.

```
:param X: list of containing all the tokens of the first string
:type X: list(str)
```

Reply:
Made the changes in `nltk.translate.util.py`